internal/util/parser_test.go

```diff
@@ -0,0 +1,80 @@
+package util
+
+import (
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+)
+
```

Comment:
I think you can either do all these in one function (with the classes testCases pattern (note that the strings could be placed in variables/constants at the end of the file to make it look cleaner),
or you can use a common function that does most of this common code.

(The first option sounds like the better one to me.)